semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/InputOps.scala

```diff
@@ -30,8 +31,14 @@ trait InputOps { self: SemanticdbOps =>
     }
 
     def isInSourceroot(sourceroot: AbsolutePath): Boolean = gsource.file match {
-      case gfile: GPlainFile =>
-        gfile.file.toPath.startsWith(config.sourceroot.toNIO)
```

Comment:
What if we keep the logic unchanged but also return true when “!gfile.file.toPath.isAbsolute”?

Reply:
Or we can also use “toAbsolutePath”

Reply:
Ahhh, that makes sense. The ` || !gfile.file.toPath.isAbsolute` works as well. Just checked. I'll push that up.